be/src/simd/selector.h

```diff
@@ -216,6 +219,157 @@ inline void avx2_select_if_common_implement(uint8_t*& selector, T*& dst, const T
         }
     }
 }
+
+#elif defined(__ARM_NEON) && defined(__aarch64__)
+template <class T>
+constexpr bool neon_could_use_common_select_if() {
+    return sizeof(T) == 1 || sizeof(T) == 2 || sizeof(T) == 4 || sizeof(T) == 8;
+}
+
+template <typename T, bool left_const = false, bool right_const = false>
+inline void neon_select_if_common_implement(uint8_t*& selector, T*& dst, const T*& a, const T*& b, int size) {
```

Comment:
why need a  `const T*&` ? can it be simplify as a `const T*` or even `const void*` ?

Reply:
> why need a `const T*&` ? can it be simplify as a `const T*` or even `const void*` ?

because outside need to handle left data if size % 16 !=0

Reply:
you can add std::enable_if here to ensure that your template won't be instantiated with unexpected types